main/src/main/scala/sbt/EvaluateTask.scala

```diff
@@ -148,18 +148,20 @@ final case class PluginData(
     resolvers: Option[Vector[Resolver]],
     report: Option[UpdateReport],
     scalacOptions: Seq[String],
+    javacOptions: Seq[String],
```

Comment:
Adding field here would break bincompat?

Reply:
That's a class that we broke already a few times. See for instance: https://github.com/sbt/sbt/pull/6553. So I think it is fine to break it again.

Should I create a 1.10.x branch?